spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -2586,6 +2587,8 @@ def stop(self, turn_off_machine=None,  # pylint: disable=arguments-differ
 
         if exc_info is not None:
             reraise(*exc_info)
+        else:
+            self.write_errored_file()
```

Comment:
Isn't this the wrong way around i.e. it will write the errored file when it *hasn't* got an error here?

Reply:
oops

Reply:
teach me for using the old version as logic point to not read. fixed